### PR TITLE
add logs, fix finding easy apply button, fix prompts for achievements

### PR DESCRIPTION
**Pull request by queukat**

### CODE REVIEW

**Comment by queukat** [comment]
add logs, 
fix finding easy apply button, 
fix checking user is logged in, 
increased steps for scrolling, 
if page is less than 3600 for scrolling, it will use the height of page
fixed chromedriver initialisation,

**Comment by feder-cr** [comment]
@queukat i like this, but this merge have a lot conflicts, please try resolve this!

**Comment by queukat** [comment]
@feder-cr sure, but I can't see pull requests in my pycharm
I did a few changes but need someone with working version control to fix lines issues (i. e., to solve the problem that different code is on the same lines)

**Comment by queukat** [comment]
@feder-cr done

File: src/linkedIn_job_manager.py [hunk 1]
```diff
@@ -75,51 +89,116 @@ def start_applying(self):
...
                 sleep_time = random.randint(50, 90)
                 utils.printyellow(f"Sleeping for {sleep_time / 60} minutes.")
+                logger.debug("Sleeping for %d seconds", sleep_time)
                 time.sleep(sleep_time)
                 page_sleep += 1
 
+    def get_jobs_from_page(self):
+        """
+        Функция для получения списка вакансий на текущей странице.
+        Если вакансии не найдены, возвращает пустой список.
```
Comment (rahul-lohra) [inline src/linkedIn_job_manager.py:143]: Why they are not in english? or internationalized?

**Review by rahul-lohra — COMMENTED** [review]
Great work man

**Pull request merged**